### PR TITLE
add note under datetime in schema reference

### LINKED ISSUE #5799: Document lack of support for zero dates in MySQL
https://github.com/prisma/prisma/issues/5006

### PR DESCRIPTION
**Pull request by grpatel**
Fixes #5799

### CODE REVIEW

File: content/200-orm/050-overview/500-databases/400-mysql.mdx [hunk 1]
```diff
@@ -127,17 +127,17 @@ The MySQL connector maps the [scalar types](/orm/prisma-schema/data-model/models
...
-| `Bytes`    | `LONGBLOB`       |
+| Prisma ORM | MySQL            | Notes                                                                       |
+| ---------- | ---------------- | --------------------------------------------------------------------------- |                            
+| `String`   | `VARCHAR(191)`   |                                                                             |
+| `Boolean`  | `BOOLEAN`        | In MySQL `BOOLEAN` is a synonym for `TINYINT(1)`                            |
+| `Int`      | `INT`            |                                                                             |
+| `BigInt`   | `BIGINT`         |                                                                             |
+| `Float`    | `DOUBLE`         |                                                                             |
+| `Decimal`  | `DECIMAL(65,30)` |                                                                             |
+| `DateTime` | `DATETIME(3)`    | Currently, we do not support zero dates (`0000-00-00`, `00:00:00`) in MySQL |
```
Comment (nikolasburk) [inline content/200-orm/050-overview/500-databases/400-mysql.mdx]: Let's write "Prisma ORM does not support" instead of "we"

**Review by nikolasburk — DISMISSED** [review]
looks great, thanks Grishma! just want small change please to write "Prisma ORM" instead of "we"

**Comment by janpio** [comment]
(Also see my comments in Slack, please)

**Comment by grpatel** [comment]
> (Also see my comments in Slack, please)

added mysql docs link and zero datetime (`0000-00-00 00:00:00`) here: [7d4a7cb](https://github.com/prisma/docs/pull/5801/commits/7d4a7cb27f17d285efc77c067f6ec95be93c09d7)

created [task](https://www.notion.so/prismaio/Docs-Expand-on-MySQL-datetime-not-supported-e51fef9d77b04ad9a623a8001c35eb87?pvs=4) to expand on this content in the future

**Review by jharrell — APPROVED** [review]

**Pull request merged**